examples/rustapi_module/tests/test_datetime.py

```diff
@@ -272,10 +272,9 @@ def test_tz_class():
     assert dt.utcoffset() == pdt.timedelta(hours=1)
     assert dt.dst() is None
 
-@pytest.mark.skip(reason='to debug with the latest master')
 def test_tz_class_introspection():
     tzi = rdt.TzClass()
 
     assert tzi.__class__ == rdt.TzClass
-    assert repr(tzi) == "TzClass()"
+    assert repr(tzi).startswith('<rustapi_module.datetime.TzClass object at')
```

Comment:
Can we not define a custom `repr` for this to make this test pass?

Or is the problem not hit if you define a custom `repr`?

Reply:
Ah, good point, though I think the `repr` could be effectively:

```python
def __repr__(self):
    return "%s()" % self.__class__.__name__
```

Your version seems fine, though.

Reply:
I'm going to take it as it because I want that change for 0.5.0, but feel free to change this into something different later